core/zms/src/main/rdl/Domain.rdli

```diff
@@ -282,3 +282,18 @@ resource DomainTemplate DELETE "/domain/{name}/template/{template}" {
         ResourceError TOO_MANY_REQUESTS;
     }
 }
+
+//List of Business Services
+type BusinessServiceList Struct {
+    Array<String> businessServices; //list of business services
+}
+
+//List all valid Business Services.
+resource BusinessServiceList GET "/bservices?user={user}" {
```

Comment:
We should also not have a specific endpoint for this - it assumes that somehow we're managing business services. We just need a more generic api if we want to implement such functionality. These are all domain meta attributes so it should be something like:

resource Array<String> GET "/domain/metastore?attribute=<attributeName>&user=<userName>

the server can verify that the attributeName is one of the supported meta store attributes (e.g. businessService, azureSubscription, awsAccount, etc. and call the appropriate megastore api.